shared-libs/rules-engine/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "test": "mocha test/*.spec.js test/**/*.spec.js",
```

Comment:
So, I have not actually been able to figure this one out yet (hence the hack here).  I got so deep into debugging this without actually being able to figure out what was going on that I wanted to just throw it out here and see if anyone else had ideas.

Basically some of the rules-engine _integration_ tests fail when the TZ is `<UTC`.  So, `UTC` and `EAT` work just fine, but `CDT`, `EST` etc fail (including the built-in `npm run tz` script here...).  For example, [this test](https://github.com/medic/cht-core/blob/0d4c275b088a92edb5b971d4f40aafcde3f93d42/shared-libs/rules-engine/test/integration.spec.js#L150) fails here [in the `triggerFacilityReminderInReadyState` function](https://github.com/medic/cht-core/blob/0d4c275b088a92edb5b971d4f40aafcde3f93d42/shared-libs/rules-engine/test/integration.spec.js#L662) because `db.bulkDocs.callCount` is `2` instead of `1`.  The extra call to `bulkDocs` is coming from a _target_ that is getting triggered in addition to the expected _task_.  

After a bunch of debugging, I have tracked the issue down to [here in the `provider-wireup`](https://github.com/medic/cht-core/blob/6514ce2fc8f5d19b2a680a1aa53355ee737129a8/shared-libs/rules-engine/src/provider-wireup.js#L299) where we get the current interval.  When running with `UTC` I get:

```json
{
  "start": 0,
  "end": 2678399999 '
}
```

but when running with `CDT` I get:

```json
{
"start": 21600000,
"end": 2699999999
}
```

This difference is caused because [the `moment()` call here](https://github.com/medic/cht-core/blob/bc97124153bf4533939281d7dd3bb050187de57c/shared-libs/calendar-interval/src/index.js#L31) is returning the moment relative to the current system TZ (`21600000` is midnight, Jan 1st 1970 CDT).